CIP-0034/README.md

```diff
@@ -0,0 +1,82 @@
+---
+CIP: 34
+Title: Smart Contract Software Licensing
+Authors: Pi Lanningham <pi@sundaeswap.finance>
+Comments-URI:
+Status: Draft
+Type: Informational
+Created: 2022-01-03
+Post-History: 
+License: CC-BY-4.0
+---
+
+## Abstract
+
+This proposal defines a metadata standard for specifying the license for a smart contract.
+
+## Motivation
+
+Smart contracts, specified in source code or compiled to raw bytecode as presented on the blockchain, are creative and often highly valuable works and are often subject to legal licensing concerns.  It is important that companies building on Cardano have the ability to link their smart contracts unambiguously to the license.
+
+This standard allows such a dApp builder to specify in the metadata of a transaction a reference to the license for each script.
+
+Ensuring that there is a consistent way to do this will have a number of benefits:
+ - Explorer projects, such as CardanoScan, can be updated to show the script license before showing the bytecode
+ - Projects have legal recourse against outright, low-effort forks of their projects (such as the Sushiswap / Uniswap fiasco)
+ - In theory, parts of the ecosystem could (at their discretion) choose to honor these licenses, and refuse to interoperate with projects that violate the license; A DEX, for example, could justify excluding a token on the basis of violation of an appropriate software license.
+
+Ultimately, the above encourages and fosters a more Open and collaborative Cardano.  With some confidence that their license-restricted code won't be copied outright, projects can feel more comfortable open-sourcing more of their intellectual property for the sake of knowledge sharing and innovation, with the risk of a whole-sale copy mitigated.
+
+## Specification
+
+This is the proposed `transaction_metadatum_label` value
+
+| transaction_metadatum_label | description            |
+| --------------------------- | ---------------------- |
+| 1998                        | Smart Contract License |
+
+This value is chosen as it is unused, and because it is the year that the [Open Source Initiative was founded](https://opensource.org/history).
+
+### Structure
+
+The structure allows for multiple smart contracts, also with different licenses, in a single transaction.  It also allows specifying licenses for only a subset of the contracts.
+
+```
+{
+  "1998": {
+    "<contract_address>": {
```

Comment:
Could we get a CDDL specification instead of a JSON specification? Transaction metadata is a subset of valid JSON so we should avoid JSON definitions